### ISSUE TITLE
Ideas to keep developing the package

### ISSUE THREAD

**Issue by ian-flores** [UBC-MDS/ColourblindR#67]
I would like to keep developing the package, here are my ideas:

- Add a theme for people acromatopsia

    - People with monochromatic vision can see no colour at all and their world consists of different shades of grey ranging from black to white, rather like only seeing the world on an old black and white television set.

- Try to add shapes that might distinguish between backgrounds like in old figures or the one attached? 

    - ![image](https://user-images.githubusercontent.com/18703558/53913031-2cb11280-400f-11e9-94ab-3448a615a231.png)

        - It's from the data viz book suggested in DSCI 532
    - This last one would be to combine the shapes with colours themes that we have already created. 

- Verify that the package works with autoplots from time series and maps

- Create functions that returns the palettes so that they can use them in combination with the leaflet package. 
    - Format: pal() and it returns a colour palette

**Comment by ian-flores** [comment]
@sabrinatkk @hfboyce

**Comment by hfboyce** [comment]
Also a grey scale palette would be useful!

**Comment by ian-flores** [comment]
Looking at the description of acromatopsia, it would fall under the grey scale palette.

Also, how about we call that one `theme_acroma()`?

**Comment by sabrinatkk** [comment]
I love this roadmap, team!

I think our priority ranking can be:

1. Theme for people with achromatopsia. I see this as a "quicker" implementation as we already have the process for the other three. It is the most impactful/useful for people within the colourblind community.

2. alter shapes for differentiation for all four themes.  

@hfboyce @ian-flores  What do you think?

**Comment by ian-flores** [comment]
I agree Sabrina. How do you want to divide the workload? I can set up the function for `theme_acroma()` and then send a PR and once someone else has the palette they can add to the PR as well

**Comment by hfboyce** [comment]
I can make the palette!!! I already have an idea of how I would design it.

**Comment by ian-flores** [comment]
@hfboyce are we going to work on this? If so, I'll keep the issue open, if not then let's do a clean up and remove issues and PR's from the settings.

**Comment by hfboyce** [comment]
I think we should keep it open!

**Comment by sabrinatkk** [comment]
Hello everyone, 

Sorry for late reply. Just had a newborn about a month ago...quite handful 

I think we should keep that open. But if you prefer cleaning it up and close it, I’ll be fine with that too.

Hope everyone is doing fine!

Regards,
Sabrina

**Comment by hfboyce** [comment]
I cannot believe you are a mom now! Baby Ethaniel is so lucky! I can't even take care of myself let alone another life!
